.github/workflows/test.yaml

```diff
@@ -18,7 +18,9 @@ jobs:
       - name: Install Bofire
         run: pip install .[testing]
       - name: Run tests
-        run: pytest -ra --cov=bofire --cov-report term-missing tests
+        run: |
+        conda install -c conda-forge cyipopt
```

Comment:
I think so far we only use linux-based runners. There you can pip install cyipopt and install ipopt via the corresponding package manager (apt).